pkg/config/config.go

```diff
@@ -45,6 +45,23 @@ type OIDC struct {
 	FriendlyNameClaimName  string            `mapstructure:"friendly_name_claim_name"`
 }
 
+// TODO(isan) consolidate with OIDC
+// CookieAuthVerification is related to auth based on a cookie set by an external service
+type CookieAuthVerification struct {
+	// ValidateIDTokenClaims if set will validate the values  (e.g department: "R&D") exist in the token claims
+	ValidateIDTokenClaims map[string]string `mapstructure:"validate_id_token_claims"`
+	// DefaultInitialGroups is a list of groups to add to the user on the lakeFS side
+	DefaultInitialGroups []string `mapstructure:"default_initial_groups"`
+	// InitialGroupsClaimName comma separated list of groups to add to the user on the lakeFS side
+	InitialGroupsClaimName string `mapstructure:"initial_groups_claim_name"`
```

Comment:
what's the difference between `DefaultInitialGroups` and `InitialGroupsClaimName` ?

Reply:
`DefaultInitialGroups` - as long as the user verified by the external system. 
`InitialGroupsClaimName ` - if provided, by the external system a claim with lakeFS groups for the user.